Resources/Prototypes/_LostParadise/Entities/Structures/Wallmounts/posters.yml

```
@@ -45,4 +45,74 @@
   - type: Sprite
     state: timetea
 
-# - >
+- type: entity
+  parent: LPPPosterBase
+  id: LPPPosterBeach
+  name: "Summer epsiode"
+  description: "It's a bit cold, but you can imagine that this picture is HOT."
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)